generated/nidaqmx/_lib_time.py

```diff
@@ -58,6 +58,12 @@ def from_datetime(cls, dt: Union[std_datetime, ht_datetime]) -> AbsoluteTime:
 
         return AbsoluteTime(lsb=lsb, msb=timestamp_1904_epoch)
 
+    def convert_to_local_timezone(self, expected_time_utc):
+        current_time_utc = ht_datetime.now(timezone.utc)
+        local_timezone_offset = current_time_utc.astimezone().utcoffset()
+        local_expected_time = expected_time_utc + local_timezone_offset
+        return local_expected_time
+
     def to_datetime(self, tzinfo: Optional[timezone] = None) -> ht_datetime:
         # First, calculate whole seconds by converting from the 1904 to 1970 epoch.
```

Comment:
Why does this function need to do anything with 1970? Adding seconds to the 1904 epoch should work before or after 1970.

Can we simplify the fraction seconds and let hightime do the heavy lifting? Does this lose precision or something?
```
_1904_EPOCH =  ht_datetime(1904, 1, 1)
...
dt = _1904_EPOCH + ht_timedelta(seconds=self.msb) + ht_timedelta(self.lsb / 2**64)
```

Reply:
Please try:
- Always using `ht_datetime(1904, 1, 1) + ht_timedelta(seconds=self.msb)`.
- Add `ht_timedelta(yoctoseconds=total_yoctoseconds)` to it. I think https://github.com/ni/hightime/blob/master/hightime/_timedelta.py should do the divmod to calculate microseconds, femtoseconds, and remainder_yoctoseconds.

Reply:
Huh, yeah, that's a lot more straight-forward. Let the `ht_datetime` class figure out all the epoch conversions. I don't know why this wouldn't work. I made it a lot harder on myself and basically reimplemented the logic those datetime classes already have for handling epochs. OOPS!